### PR TITLE
Fix alternative recipient

### PR DESCRIPTION
**Pull request by Mi-Lan**
related sdk pr https://github.com/levelkdev/swapr-sdk/pull/228
- Fixed alternative recipient for uniswapTrade and GnosisTrade
- There is no clear solution for the 0xTrade to accept alternative recipient since their api doesn't support it at the moment
- I just disabled alternative recipient when 0xTrade is selected @zamli maybe we can mark it red and add additional message when trade doesn't support alternative recipient
- cleaned up code abit

### CODE REVIEW

**Review by MilanVojnovic95 — APPROVED** [review]
Tested and looks ok to me 👍

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "@reduxjs/toolkit": "^1.8.3",
     "@swapr/core": "^0.3.19",
     "@swapr/periphery": "^0.3.22",
-    "@swapr/sdk": "1.7.0-beta.0",
+    "@swapr/sdk": "https://git@github.com/levelkdev/swapr-sdk.git#8c763bc87336352dbff3e9b8ac4d8310badbede9",
```
Comment (berteotti) [inline package.json:45]: PR looks good, will approve when this is pointing to a new sdk release

File: src/pages/Swap/LimitOrderBox/api/cow.ts [hunk 2]
```diff
@@ -33,23 +33,41 @@ type GetLimitOrderQuoteParams = SignLimitOrderParams
...
+      buyToken,
+      sellAmountBeforeFee: sellAmount,
+      from: userAddress,
+      kind: CoWOrderKind.SELL,
+      partiallyFillable: false,
+      receiver: receiverAddress,
+      sellToken,
+      validTo: expiresAt,
+    })
+  }
```
Comment (Mi-Lan) [inline src/pages/Swap/LimitOrderBox/api/cow.ts:70]: @adamazad I guess I should have expected code sabotage when I saw your name on the commit :D
Screenshot 2022-12-27 at 17 17 58

Comment (adamazad) [reply src/pages/Swap/LimitOrderBox/api/cow.ts:70]: I explained on the standup. See https://github.com/cowprotocol/cow-sdk/issues/91

**Review by adamazad — APPROVED** [review]
COW worked,

https://explorer.cow.fi/xdai/orders/0x0ce06dc2404a445a98cc0f50d01bb2f9ee75ef0f39baa5253330d2d58ec30ddef17bbf8ce0e4b3fd216a659bb15199f877aad6fd63ac2a62


For Curve on GC, we need to go back to the SDK later on: https://gnosisscan.io/tx/0x09d17f91051a943568c4e25dceca9a8d565daa88ed868ba6fb423ff64632b9a3

**Pull request merged**